shell/src/main/java/alluxio/cli/GetConf.java

```diff
@@ -14,22 +14,49 @@
 import alluxio.Configuration;
 import alluxio.PropertyKey;
 
+import com.google.common.base.Preconditions;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
 import java.util.Map.Entry;
 
 /**
  * Utility for printing Alluxio configuration.
  */
 public final class GetConf {
-  private static final String USAGE = "USAGE: GetConf [KEY]\n\n"
+  private static final String USAGE = "USAGE: GetConf [-bytes] [KEY]\n\n"
       + "GetConf [KEY] prints the configured value for the given key. If the key is invalid, the "
       + "exit code will be nonzero. If the key is valid but isn't set, an empty string is printed. "
-      + "If no key is specified, all configuration is printed.";
+      + "If no key is specified, all configuration is printed. If \"bytes\" option is specified, "
+      + "a value of \"1KB\" will be converted to a value of 1024.";
+
+  private static final String BYTES_OPTION_NAME = "bytes";
```

Comment:
as discussed offline, let's change this flag to `unit` which accepts B, KB, MB, GB, TB ... and optionally introduce another flag called `quantity` which accepts `size` ... this gives us the affordance to for instance introduce `time` as another quantity in the future, with supported units ns, us, ms, s, m, h, d, ...